magento/pwa-studio
# [PWA-595] GraphQL Build Warning

## Description

During build our graphql validation script is throwing the following warning:

useEditableForm.js
Warning: fragment with name SelectedShippingMethodFragment already exists.
graphql-tag enforces all fragment names across your application to be unique; read more about this in the docs: http://dev.apollodata.com/core/fragments.html#unique-names

## Related Issue





* [[PWA-595](https://jira.corp.magento.com/browse/PWA-595)] GraphQL Build Warning

## Acceptance 



### Verification Stakeholders

### Specification



### Verification Steps


1. `yarn build`
2. Verify warning about fragments are no longer displayed

This change touched both Cart and Checkout experiences for Shipping Methods, so test all areas intersecting with Shipping Method selection.

Cart
1. Add items to cart and navigate to `/cart`
2. Estimate your shipping
3. Verify that you can select methods and selection is displayed in order summary

Checkout
1. Add items to cart and navigate to `/checkout`
2. Enter shipping information
3. Select shipping method
4. Verify selection is displayed in order summary

## Screenshots / Screen Captures (if appropriate)

## Checklist

* I have updated the documentation accordingly, if necessary.
[ ] I have added tests to cover my changes, if necessary.
